File: app/src/main/java/tpcreative/co/qrscanner/model/ItemNavigation.java
package tpcreative.co.qrscanner.model;

import com.google.zxing.client.result.ParsedResultType;

import java.io.Serializable;

public class ItemNavigation implements Serializable {
    public int res;
    public EnumFragmentType enumFragmentType;
    public String value;
    public EnumAction enumAction;
    public ParsedResultType resultType;


    public ItemNavigation(ParsedResultType parsedResultType,EnumFragmentType enumFragmentType, EnumAction enumAction,int res, String value){
        this.enumFragmentType =enumFragmentType;
        this.enumAction = enumAction;
        this.value = value;
        this.res = res;
        this.resultType = parsedResultType;
    }

}

File: app/src/main/java/tpcreative/co/qrscanner/model/ItemNavigation.kt
package tpcreative.co.qrscanner.model
import com.google.zxing.BarcodeFormat
import com.google.zxing.client.result.ParsedResultType
import java.io.Serializable

class ItemNavigation(var resultType: ParsedResultType?,val contactKey: String,val contactValue : String,var barcodeFormat: String?, var enumFragmentType: EnumFragmentType?, var enumAction: EnumAction?, var res: Int, var value: String?, var isFavorite : Boolean?) : Serializable